cmd/client/main.go

```diff
@@ -46,6 +47,11 @@ func createBootstrapCommand() *cobra.Command {
 			}
 
 			logger := client.GetLogger(opts.LogFormat, opts.Verbose)
+			defer func() {
+				if err := logger.Sync(); err != nil {
```

Comment:
could we potentially export this to a helper func, then defer a call to that instead?